package.json

```diff
@@ -83,9 +87,10 @@
     "eslint-plugin-istanbul": "0.1.2",
     "eslint-plugin-node": "11.1.0",
     "express": "4.17.1",
-    "graphiql": "1.0.6",
+    "graphiql": "^1.4.1",
```

Comment:
👍 

Currently you can't provide request headers when using graphiql, getting this updated version in here would be very helpful.